.github/workflows/ci.yml

```diff
@@ -253,27 +249,21 @@ jobs:
               asv machine --yes
 
               # check Modin on Ray
-              asv run --quick --strict --show-stderr --launch-method=spawn \
+              asv run --quick --show-stderr --launch-method=spawn \
```

Comment:
what did `--strict` do and why do you want to remove it?

Reply:
reverted